config/locales/views/en.yml

```
@@ -678,6 +678,10 @@ en:
           hide_content_for_others: hide their works, series, bookmarks, and comments from anyone else
           intro: 'Muting a user will not:'
           prevent_emails: prevent you from receiving comment or subscription emails from this user
+  orphans:
+    orphan_user:
+      orphaning_bylines_only_message_html: Orphaning will automatically remove your personal information from <strong>bylines only.</strong> It will <strong>not</strong> automatically remove your personal information from anywhere else in the work(s). Social media accounts, contact information, email addresses, names, and any other personal information posted in the title, summary, notes, tags, work body, or comment text will <strong>not</strong> be automatically removed. Comments posted by other users will also <strong>not</strong> be affected by Orphaning. If you want information removed from these places, you should edit or delete it prior to Orphaning. <strong>Once you Orphan the work(s), you will no longer be able to delete information in these places yourself.</strong>
+      orphaning_works_message_html: 'Orphaning all of your works will <strong>permanently</strong> remove your username and/or pseud from the bylines of: the following work(s), their chapters, associated series, and any comments you may have left on them while logged into this account.'
```

Per Jira this should be:
 ```suggestion
       orphaning_works_message_html: 'Orphaning will <strong>permanently</strong> remove your username and/or pseud from the bylines of: the following work(s), their chapters, associated series, and any comments you may have left on them while logged into this account.'
```

---

Good catch, thank you. Addressed.